android/MobiledgeXSDKDemo/README.md

```diff
@@ -16,80 +16,81 @@
 - Predictive Quality of Service
 
 ### Cloudlet Map
-The app's startup page is a world map showing MobiledgeX cloudlets where the app's backend is running. If no cloudlets are showing, or you don't see the set cloudlets you expect to see, click the main menu "hamburger" icon, and select **Settings**, then **General Settings**. From there, select the **Region** and **Operator Name** you're interested in.
+The app's startup page is a world map showing MobiledgeX cloudlets where the app's backend is running. If no cloudlets are shown, or you don't see the set cloudlets you expect to see, click the main menu "hamburger" icon and select **Settings**, then **General Settings**. From here, select the **Region** and **Operator Name** you're interested in.
 
-Tap on any of the cloudlets, and a panel with the cloudlet's name will appear. On that panel, you can "click for details", to see cloudlet information like latitude, longitude, and distance from your location.
+Tap on any of the cloudlets, and a panel with the cloudlet's name will appear. On that panel, you can "click for details" to see cloudlet information like latitude, longitude, and distance from your location.
 
 #### Cloudlet Details
-In addition to showing details about the cloudlet, this page also lets you initiate a latency test, or a download speed test. The settings for this page let you control things like the number of packets for the latency test, or the size of the download. Click the "Gear" icon to access these settings.
+In addition to showing the details of the cloudlet, this page allows the initiation of a latency test or a download speed test. The settings of this page, allow control of conditions, such as the number of packets for the latency test, or the size of the download. Click the "Gear" icon to access these settings.
 
-You can also choose this cloudlet to be used as the **Edge** server for Face Detection and Face Recognition. Tap the 3-dot menu and select "Use as Face Recognition Edge Host". The **Cloud** server can be set in the same way.
+YYou can also choose this cloudlet to be used as the **Edge** server for Face Detection and Face Recognition. Tap the 3-dot menu and select "Use as Face Recognition Edge Host". The **Cloud** server can be configured in the same way.
 
 ### SDK Calls
 #### Register Client
-This will call [RegisterClient](http://swagger.mobiledgex.net/client/#operation/RegisterClient "RegisterClient") with information that identifies this app's backend software. The session cookie is shown to verify that the call was successful.
+SDK Calls summon the [RegisterClientAPI](http://swagger.mobiledgex.net/client/#operation/RegisterClient "RegisterClient") with information that identifies this app's backend software. The session cookie is shown to verify that the call was successful.
```

Comment:
"SDK Calls summon the" is pretty awkward. I would suggest making it like the others below. "Register Client will call".

Reply:
I changed the word from "call" to "summon" to break up the repetitive use that word and to encourage user engagement.

Reply:
But it is literally a "function call" or "SDK call". The plural "SDK Calls" would be fine for an overview of the different calls, but this line refers to a singular "call".
"XYZ will call" is repeated 4 times below, which I think is fine for this kind of doc.